homeassistant/components/traccar/device_tracker.py

```diff
@@ -91,6 +76,7 @@
     scanner = TraccarScanner(
         api, hass, async_see,
         config.get(CONF_SCAN_INTERVAL, SCAN_INTERVAL),
+        config.get(CONF_MAX_ACCURACY), config[CONF_SKIP_ACCURACY_ON],
```

Comment:
```suggestion
        config[CONF_MAX_ACCURACY], config[CONF_SKIP_ACCURACY_ON],
```

Use [] syntax for keys that are in the validated data.